hardware/src/base-station-sketch2/sketch_aug16a.ino

```diff
@@ -0,0 +1,55 @@
+#include "CmdMessenger.h"
+
+/* Define available CmdMessenger commands */
+enum {
+    current_status,
+    status_is,
+    think_for_me,
+    thought_is,
+    error,
+};
+
+/* Initialize CmdMessenger -- this should match PyCmdMessenger instance */
+const int BAUD_RATE = 9600;
+CmdMessenger c = CmdMessenger(Serial,',',';','/');
```

Comment:
Looked at the sauce for the constructor Def in ```CmdMessenger.h``` and they're actually unnecessary in this case. Remove all arguments other than ```Serial``` and it should work the same